Userland/Libraries/LibGL/Buffer/Buffer.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022, cflip <cflip@cflip.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Buffer.h"
+#include <AK/Format.h>
```

Comment:
Please order alphabetically

Reply:
Turns out I didn't even need `AK/Format.h` in the first place!